TeensyDmx.h

```diff
@@ -117,6 +117,13 @@ class TeensyDmx
     bool newFrame();
     // Use for receive
     const volatile uint8_t* getBuffer() const;
+    // Use for receive with addresses from 0-511
+    uint8_t getChannel(const uint16_t address, const uint8_t value);
```

Comment:
Unrequired and unused parameter value.

Reply:
Oops, careless copy and paste error fixed!